tests/unit/conftest.py

```diff
@@ -137,13 +138,19 @@ def pyramid_request(db_session, application_instance, lti_v11_params):
 
 
 @pytest.fixture
-def user_is_learner(pyramid_request):
-    pyramid_request.lti_user.roles = "Learner"
+def user_is_learner(pyramid_request, lti_user):
```

Comment:
This now changes both `lti_user` and `pyramid_request`

Should this be two fixtures? In general the fixtures in this file try to line everything up which is pretty nice when that what you want to test but force you to use a new in some cases.

I reckon when using `user_is_learner` in the current code base I'd expect the request and the user to line up so that's the change.

Reply:
Yeah, it seems to me like `user_is_learner` ought to modify the Pyramid request's actual user, so this makes sense :+1: